### PR TITLE
Fixed: when it has no files to diff

### PR DESCRIPTION
**Pull request by krazyeom**
parent.prevFocused.selected return 0, even the list has no file

### CODE REVIEW

File: controller/main.js [hunk 1]
```diff
@@ -193,7 +193,11 @@ _.each(view.list, function (elem) {
   //elem.unkey(['C-d']);
 
   elem.key(['C-d'], function () {
-    main.show(diff);
+    var canDiff = view.list.unstaged.interactive ? git.files.unstaged.length : git.files.staged.length;
+
+    if (canDiff) {
+      main.show(diff);
+    }
   });
```
Comment (golbin) [inline controller/main.js:202]: I think, it's better the checking codes will be in **controller/diff.js**

Add below codes before the line 8 in the diff.js;

``` js
if (parent.prevFocused.selected === undefined) {
  return;
}
```

It's cleaner and makes easy to refactor. :-) (something like adding error messages box)

Comment (golbin) [inline controller/main.js:202]: OR, you can add below codes before line 196 in this file. :-)

``` js
if (view.screen.focused.selected === undefined) {
  return;
}
```

**Comment by krazyeom** [comment]
yes, it's right.

**Comment by golbin** [comment]
Above codes have a bug. Runable codes are below;

``` js
if (this.getItem(this.selected) === undefined) {                      
  return;                                                           
}
```

**Pull request closed without merging**